app/Http/Controllers/WebhookController.php

```diff
@@ -83,9 +88,22 @@ public function edit(Webhook $webhook)
      * @param  \App\Models\Webhook  $webhook
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, Webhook $webhook)
+    public function update(WebhookUpdateRequest $request)
     {
-        //
+        $data = $request->except('_token');
+        $check_webhook = $this->webhookRepository->getWebhookByName($data['name']);
+        if(isset($check_webhook)) {
+            if($check_webhook->id != $data['id']){
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 96</br>Variable "check_webhook" is not in valid camel caps format